R/tm_a_pca.R

```diff
@@ -165,7 +165,14 @@ tm_a_pca <- function(label = "Principal Component Analysis",
   checkmate::assert_multi_class(pre_output, c("shiny.tag", "shiny.tag.list", "html"), null.ok = TRUE)
   checkmate::assert_multi_class(post_output, c("shiny.tag", "shiny.tag.list", "html"), null.ok = TRUE)
 
-  checkmate::assert_list(decorators, "teal_transform_module", null.ok = TRUE)
+  if (checkmate::test_list(decorators, "teal_transform_module", null.ok = TRUE)) {
+    decorators <- if (checkmate::test_names(names(decorators), subset.of = c("default", "plot"))) {
+      lapply(decorators, list)
+    } else {
+      list(default = decorators)
+    }
+  }
+  assert_decorators(decorators, null.ok = TRUE, names = c("default", "plot"))
```

Comment:
Not yet specified anywhere `assert_decorators`

Reply:
It is in `utils.R`

https://github.com/insightsengineering/teal.modules.general/blob/7a96181036177a8bda62315655a93c313f9495d1/R/utils.R#L376-L379

Reply:
thanks, sorry, I must have missed it